### PR TITLE
Add NetworkPolicy to Redis

### PR DESCRIPTION
**Pull request by ozdanborne**
cc: @lachie83

### CODE REVIEW

**Comment by lachie83** [comment]
cc @prydonius

File: stable/redis/README.md [hunk 1]
```diff
@@ -84,6 +86,21 @@ $ helm install --name my-release -f values.yaml stable/redis
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+## NetworkPolicy
+
+To enable network policy for redis, install
+[a networking plugin that implements the Kubernetes NetworkPolicy spec](https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy#before-you-begin),
+and set `NetworkPolicy.Enabled` to `true`.
```
Comment (prydonius) [inline stable/redis/README.md]: `networkPolicy.enabled`

File: stable/redis/README.md [hunk 2]
```diff
@@ -84,6 +86,21 @@ $ helm install --name my-release -f values.yaml stable/redis
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+## NetworkPolicy
+
+To enable network policy for redis, install
```
Comment (prydonius) [inline stable/redis/README.md]: for consistency, can we use `Redis` instead of `redis`?

File: stable/redis/README.md [hunk 3]
```diff
@@ -84,6 +86,21 @@ $ helm install --name my-release -f values.yaml stable/redis
...
+[a networking plugin that implements the Kubernetes NetworkPolicy spec](https://kubernetes.io/docs/tasks/administer-cluster/declare-network-policy#before-you-begin),
+and set `NetworkPolicy.Enabled` to `true`.
+
+For Kubernetes v1.5 & v1.6, you must also turn on NetworkPolicy by setting
+the DefaultDeny namespace annotation. Note: this will enforce policy for _all_ pods in the namespace:
+
+    kubectl annotate namespace default "net.beta.kubernetes.io/network-policy={\"ingress\":{\"isolation\":\"DefaultDeny\"}}"
+
+With NetworkPolicy enabled, only pods with the generated client label will be
+able to connect to redis. This label will be displayed in the output
```
Comment (prydonius) [inline stable/redis/README.md]: capitalize redis here too

**Review by prydonius — COMMENTED** [review]
awesome, thanks @ozdanborne, just have some quick README comments

**Comment by ozdanborne** [comment]
@prydonius thanks for the review. I put a few more changes in a second commit that I noticed were needed. I'll squash before we're ready to merge

**Comment by prydonius** [comment]
/lgtm

thanks @ozdanborne, no need for a squash, we usually use GitHub's squash and merge feature

**Pull request merged**